java/test/jmri/jmrit/display/controlPanelEditor/shape/DrawCircleTest.java

```diff
@@ -15,21 +16,22 @@
  */
 public class DrawCircleTest {
 
+    EditorScaffold editor;
+
     @Test
     public void testCTor() {
         Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         ControlPanelEditor frame = new ControlPanelEditor();
-        DrawCircle t = new DrawCircle("newShape", "Circle", null);
+        DrawCircle t = new DrawCircle("newShape", "Circle", null, editor, false);
         Assert.assertNotNull("exists", t);
         JUnitUtil.dispose(t);
         JUnitUtil.dispose(frame);
     }
 
     public void testCTorEdit() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
-        ControlPanelEditor frame = new ControlPanelEditor();
+       ControlPanelEditor frame = new ControlPanelEditor();
         PositionableCircle ps =  new PositionableCircle(frame);
-        DrawRectangle t = new DrawRoundRect("editShape", "Circle", ps);
+        DrawRectangle t = new DrawRoundRect("editShape", "Circle", ps, editor, true);
```

Comment:
There are several reasons this is failing.

1) This PR removes the graphicsEnvironment check.  That needs to be restored.
2) The constructor tests are failing is that the new constructor uses an editor to display the shapes.  The editor passed is an EditorScaffold object which isn’t intended to be displayed.  There is also a ControlPanelEditor created.  This should be passed instead of the EditorScafold.  The EditorScaffold object was added in this PR.